src/beignet/nn/functional/_torsion_angle_loss.py

```diff
@@ -0,0 +1,36 @@
+from typing import Tuple
```

Comment:
We could use [PEP 585 ](https://peps.python.org/pep-0585/) instead so the hint would be `tuple[Tensor, Tensor]`. I find it slightly cleaner but not a big deal either way.

Reply:
Looking into this more it looks like those hints are actually being deprecated and removed once python 3.9 is EOL